### DISCUSSION TITLE
如何暂时禁止nvim-lsp-installer的更新检查

### DISCUSSION THREAD

**Discussion by allworldg** [nshen/learn-neovim-lua#52] [Q&A] [answered]
每次使用LSP,Installer都会自动检查server更新状态，有什么办法能禁止吗？类似vscode插件禁止自动检查更新那样？

**Comment by nshen** [comment, marked as answer]
会自动检查嘛？ LspInstallInfo 命令才会检查 server更新状态吧？

**Reply by allworldg** [reply]
哦就是那个命令，我以为LspInstallInfo只是查看下载成不成功而已

**Reply by nshen** [reply]
他只是查一下是否有新版本的lsp，应该不影响使用吧

**Reply by allworldg** [reply]
好的，谢谢
